var/spack/repos/builtin/packages/py-sina/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PySina(PythonPackage):
+    """Sina allows codes to store, query, and visualize their data through an
+    easy-to-use Python API. Data that fits its recognized schema can be ingested
+    into one or more supported backends.
+    Sina's API is independent of backend and gives users the benefits of a database
+    without requiring knowledge of one, allowing queries to be expressed in pure
+    Python.  Visualizations are also provided through Python.
+
+    Sina is intended especially for use with run metadata,
+    allowing users to easily and efficiently find simulation runs that match some
+    criteria.
+    """
+
+    homepage = "https://github.com/LLNL/Sina"
+    git = "https://github.com/LLNL/Sina"
+
+    # notify when the package is updated.
+    maintainers = [
+        'HaluskaR',
+        'estebanpauli',
+        'murray55',
+        'doutriaux1',
+    ]
+
+    version('1.11.0', tag="v1.11.0")
+    version('1.10.0', tag="v1.10.0")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-ujson', type=('build', 'run'))
+    depends_on("py-sqlalchemy", type="run")
+    depends_on("py-six", type="run")
```

Comment:
```suggestion
    depends_on("py-sqlalchemy", type=("build", "run"))
    depends_on("py-six", type=("build", "run"))
```
You're also missing the following deps:
```python
        'enum34;python_version<"3.4"',                                                   
        'orjson;python_version>="3.6" and platform_machine!="ppc64le"',                  
        'ujson;python_version>="3.6" and platform_machine=="ppc64le"',                   
        'ujson<4;python_version<"3.6" and platform_machine!="ppc64le"',         
```

Reply:
how would I specify enum34;python_version<"3.4 in spack?
orjson does not exists in spack so we have to use ujson everywhere.

Reply:
> how would I specify enum34;python_version<"3.4 in spack?

```python
depends_on('py-enum34', when='^python@:3.3', type=('build', 'run'))
```

> orjson does not exists in spack so we have to use ujson everywhere.

I'm not sure if that will work unless you're on ppc64le. It shouldn't be too hard to add a `orjson` package.

Reply:
I'm positive orjson is only a "nice to have" json package (it's a bit faster). orjson uses not so common build tools "rust" and "maturin". If it ever makes it into spack I'm willing to update the recipe to use it.

Reply:
building on CZ i now get this:
==> Bootstrapping clingo from sources
==> Error: cannot bootstrap the "clingo" Python module from spec "clingo-bootstrap@spack+python %gcc target=x86_64" due to the following failures:
    'github-actions' raised FetchCacheError: Multiple errors during fetching:
        Error 1: RuntimeError: Unable to read index hash https://mirror.spack.io/bootstrap/github-actions/v0.1/build_cache/index.json.hash due to SpackWebError: Download failed: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1091)>
        Error 2: RuntimeError: Unable to read index https://mirror.spack.io/bootstrap/github-actions/v0.1/build_cache/index.json due to SpackWebError: Download failed: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1091)>
    'spack-install' raised ValueError: Invalid config scope: 'bootstrap'.  Must be one of odict_keys(['_builtin', 'defaults', 'defaults/linux', 'bootstrap/linux', 'disable_modules', 'overrides-0'])
    Please run `spack -d spec zlib` for more verbose error messages

Reply:
@alalazo can you help debug this bootstrap error?

Reply:
@alalazo I'm still getting this error today.

Reply:
@alalazo moving from quartz to surface, it now works fine. Not sure what's up with quartz....